src/site/apt/troubleshooting.apt

```diff
@@ -0,0 +1,14 @@
+Troubleshooting
+
+ Standard project-build-plugin errors
+
+*-----*------*------*
+<<Error Message>>		|		<<Cause>> 		| 		<<Possible Fix>>
+*-----*------*------*
+ InvocationTargetException: No class loader avaible for bundle: ch.ivyteam.ivy.dataclasses.build    | Maven uses wrong Java version	|	Change your Java version if you have multiple versions (export JAVA_HOME=</your/Jre/Dir>)  
+*-----*------*------*
+ adittionalVmOptions not getting initialized														| Multiple additionalVmOptions | Make sure you're on the latest version of project-build-plugin if it doesn't help and you have multiple adittionalVmOptions add a whitespace at the end of each argument
+*-----*------*------*
+ InstanceAlreadyExistsException  																	| Multiple project-build-plugin versions declared | Make sure that you have only one project-build-plugin version declared if you have multiple pom.xml files
```

Comment:
this is the most tricky and important issue for me: therefore I'd extend it with an example:
e.g. how about an extract from a parent POM that contains a project-build-plugin definition for the basic build/test lifecycle .... but now a child POM also defines this plugin (in different version) to define integration test behaviour. 
++ I know that this is a lot for this FAQ section. So I'd setup a fake question & answer in answers.axonivy.com and like it here...

Reply:
Maybe we could even completely switch into a Q&A based solution direction:
- means document all that stuff in Q&A : so that we can enrich the problem with reasonable examples.
- then here we could link/list programatically all q&A entries that come with a tag (e.g. project-build-plugin).
.... just an idea - I'm also agree to the current approach taken here. So let's discuss the best solution in time.